<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>691</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmd_port_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string>COM1</string>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="Yellow">
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Green">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="YELLOW_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>79</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>YELLOW_1</string>
    </property>
   </widget>
   <widget class="QLabel" name="GREEN">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>79</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="RED">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QLabel" name="YELLOW_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>YELLOW_2</string>
    </property>
   </widget>
   <widget class="QLabel" name="WHITE">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>WHITE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>正线</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>侧线</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>104</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>21</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="rail3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
